yarn.lock

```diff
@@ -15188,6 +15188,11 @@ node-gyp@^8.2.0:
     tar "^6.1.2"
     which "^2.0.2"
 
+node-lcs@^1.0.4:
```

Comment:
node-lcs has [no dependencies](https://www.npmjs.com/package/node-lcs?activeTab=dependencies), so it should be a pretty lightweight include.